README.md

```diff
@@ -4,7 +4,7 @@
 
 Get inside your stronghold and make all your Django views default login_required
 
-Stronghold is a very small and easy to use django app that makes all your Django project default to require login for all of your views.
+Stronghold is a very small and easy to use Django app that makes your project require login on all of your views.
```

Comment:
I've got a improvement to your improvement :) This changes the default behaviour so we should keep driving that home. 

`Stonghold is a very small and easy to use Django app that defaults all your project's views to require login.`